src/main/java/com/enderio/base/common/tag/EIOTags.java

```diff
@@ -94,6 +94,7 @@ private static void init() {}
 
         public static final TagKey<Fluid> COLD_FIRE_IGNITER_FUEL = FluidTags.create(EnderIO.loc("fluid_fuel/cold_fire_igniter"));
         public static final TagKey<Fluid> STAFF_OF_LEVITY_FUEL = FluidTags.create(EnderIO.loc("fluid_fuel/staff_of_levity"));
+        public static final TagKey<Fluid> EXPERIENCE = FluidTags.create(new ResourceLocation("forge", "fluid/experience")); //TODO find tag?
```

Comment:
the tag used by cyclic is `forge:experience`, we should use that as well

Reply:
Hmm this is actually more complex maybe. If the tank has one type of experience fluid, will it allow another to be "merged" into it? Should we make this happen? The safest is to only use our own experience though.

Reply:
it would be the safest, but I think it should accept all of them

Reply:
The thing is that it would need some special code and care, since it would need to ignore the (stack) fluid and still fill when it's not the same. And than we get to the issue of what to return inside the tank. *And possible mods using different units...*

Reply:
mods using different amounts for the same thing is not the case I think. And if so a standard would be better, because why should the tag exist if it's not meant for equality